modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AbstractCSharpCodegen.java

```diff
@@ -492,14 +492,6 @@ public void postProcessModelProperty(CodegenModel model, CodegenProperty propert
     public ModelsMap postProcessModels(ModelsMap objs) {
         for (ModelMap mo : objs.getModels()) {
             CodegenModel cm = mo.getModel();
-            for (CodegenProperty var : cm.vars) {
-                // check to see if model name is same as the property name
-                // which will result in compilation error
-                // if found, prepend with _ to workaround the limitation
-                if (var.name.equalsIgnoreCase(cm.classname)) {
-                    var.name = "_" + var.name;
-                }
-            }
```

Comment:
I think we may need to keep this code block and replace "_" with "var_" instead. We'll see after merging this PR.

Reply:
Appveyor CI tests passed. Confirmed there's no issue.